public/pages/CreateTransform/components/DefineTransforms/DefineTransforms.tsx

```diff
@@ -150,8 +150,13 @@ export default function DefineTransforms({
         return data[rowIndex]._source[correspondingTextColumnId] ? data[rowIndex]._source[correspondingTextColumnId] : "-";
       } else if (columns?.find((column) => column.id == columnId).schema == "date") {
         return data[rowIndex]._source[columnId] ? renderTime(data[rowIndex]._source[columnId]) : "-";
+      } else if (columns?.find((column) => column.id == columnId).schema == "geo_point") {
+        return data[rowIndex]._source[columnId].lat + ", " + data[rowIndex]._source[columnId].lon ?
+          data[rowIndex]._source[columnId].lat + ", " + data[rowIndex]._source[columnId].lon : "-";
+      } else if (columns?.find((column) => column.id == columnId).schema == "boolean") {
+        return data[rowIndex]._source[columnId] == null ? "-" : (data[rowIndex]._source[columnId] ? "true" : "false");
       }
-      return data[rowIndex]._source[columnId] ? data[rowIndex]._source[columnId] : "-";
+      return JSON.stringify(data[rowIndex]._source[columnId]) ? JSON.stringify(data[rowIndex]._source[columnId]) : "-";
```

Comment:
this seems incorrect shouldn't this be? `data[rowIndex]._source[columnId]) ? JSON.stringify(data[rowIndex]._source[columnId]) : "-";`